packages/react/src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -20,6 +15,8 @@ import {
   FloatingFocusManager,
 } from '@floating-ui/react';
 
+import { useIsomorphicLayoutEffect } from '../../hooks/useIsomorphicLayoutEffect';
```

Comment:
Was it deliberate to use `../../hooks/useIsomorphicLayoutEffect` instead of `../../hooks`?

Reply:
No, auto import decided this

Reply:
aight